docs/setup/configuring-logging.asciidoc

```diff
@@ -21,11 +21,11 @@ __<<logging-layouts,Layouts>>__ define how log messages are formatted and what t
 [[log-level]]
 === Log level
 
-Currently we support the following log levels: _all_, _fatal_, _error_, _warn_, _info_, _debug_, _trace_, _off_.
+Currently we support the following log levels: _off_, _fatal_, _error_, _warn_, _info_, _debug_, _trace_, _all_.
 
-Levels are ordered, so _all_ > _fatal_ > _error_ > _warn_ > _info_ > _debug_ > _trace_ > _off_.
+Levels are ordered, so _off_ > _fatal_ > _error_ > _warn_ > _info_ > _debug_ > _trace_ > _all_.
```

Comment:
Not correct.

Our [internal docs](https://docs.elastic.dev/kibana-dev-docs/services/logging#log-level) have the order as

> Levels are ordered, so all > fatal > error > warn > info > debug > trace > off.

Nothing has changed for logging level (tested in dev).
Example:
for a specific logger, if the level is `all`, then changed to `info`, logging is at the `info` level, not `all`.
Hence `all` > `info`.

Reply:
> Example:
> for a specific logger, if the level is all, then changed to info, logging is at the info level, not all.
Hence all > info.

I apologize if this is just me being dense, but I still find this example very confusing, particularly the conclusion. Is there a _property_ that we can reference in this comparison? E.g.

> If the log level is `error`, records at the `all`, `fatal`,  and `error` levels will be logged as they have <i>equal or greater <b>precedence</b></i> to `error`

"Precedence" might not be the correct word here, but I hope my point is illustrated.

I think that my confusion with 

` _all_ > _fatal_ > _error_ > _warn_ > _info_ > _debug_ > _trace_ > _off_` 

Is that I was interpreting the levels as indicative of _verbosity_, rather than of _priority_ (in which case `all` was most verbose, and `off` least verbose). This also explains my confusion with the comparators.